components/types/events.ts

```diff
@@ -153,6 +153,7 @@ export type PacifyC2SEvent = KillC2SEvent
 
 export type MurderedBodySeenC2SEvent = ClientToServerEvent<{
     Witness: string
+    /** `true` iff the witness is a target */
```

Comment:
```suggestion
    /** `true` if the witness is a target */
```

Reply:
The double f was actually intentional here, but I guess it does look a little weird